### ISSUE TITLE
Integration screwed up after March 2021

### ISSUE THREAD

**Issue by LouiS22** [alextud/ha-switchbot-curtain#29]
Hello there,

Unfortunately the integration is somehow broken after March 7 2021 - that's the last commit where the curtain's are working (at least for me, running on an NUC, but I've seen plenty of complaints where the integration went offline, bluetooth went offline, etc). I Don't know what changed there and then (I suspect it has something to do with the bluetooth stack.

**Comment by robimac04** [comment]
Yeah, same for me - I just manually rolled back to d029b31 and it's working fine again...

**Comment by niemyjski** [comment]
@alextud can we please get these issues addressed :)
